FastSimulation/Configuration/python/FamosSequences_cff.py

```diff
@@ -203,7 +203,21 @@
 electronGsfTracks.TTRHBuilder = 'WithoutRefit'
 electronGsfTracks.TrajectoryInEvent = True
 
-
+from RecoParticleFlow.PFTracking.trackerDrivenElectronSeeds_cff import *
+from RecoParticleFlow.PFTracking.mergedElectronSeeds_cfi import *
+
+trackerDrivenElectronSeedsTmp = trackerDrivenElectronSeeds.clone()
+trackerDrivenElectronSeedsTmp.TkColList = cms.VInputTag(cms.InputTag("generalTracksBeforeMixing"))
+trackerDrivenElectronSeeds = cms.EDProducer(
+    "ElectronSeedTrackRefFix",
+    PreGsfLabel = trackerDrivenElectronSeedsTmp.PreGsfLabel,
+    PreIdLabel = trackerDrivenElectronSeedsTmp.PreIdLabel,
+    oldTrackCollection = trackerDrivenElectronSeedsTmp.TkColList[0],
+    newTrackCollection = cms.InputTag("generalTracks"),
+    seedCollection = cms.InputTag("trackerDrivenElectronSeedsTmp",trackerDrivenElectronSeedsTmp.PreGsfLabel.value()),
+    idCollection = cms.InputTag("trackerDrivenElectronSeedsTmp",trackerDrivenElectronSeedsTmp.PreIdLabel.value())
+    )
```

Comment:
@lveldere - sorry, I had missed the redefinition of trackerDrivenElectronSeeds here before. Is it needed or is it just the same as in the trackerDrivenElectronSeeds_cff?